tools/dart-api-builder/services/packageInfo.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('canonical-path');
+
+/**
+ * Load information about this project from the pubspec.yaml
+ * @return {Object} The package information
+ */
+module.exports = function packageInfo() {
+  const ngPath = '../angular-dart';
```

Comment:
This is hard coded here but is a constant in the index.js file. Perhaps it should be passed as some kind of property or dependency?